perf/jet_test_nfailures.jl

```diff
@@ -26,7 +26,7 @@ using Test
     # inference. By increasing this counter, we acknowledge that
     # we have introduced an inference failure. We hope to drive
     # this number down to 0.
-    n_allowed_failures = 84
+    n_allowed_failures = 844
```

Comment:
The increase in `n_allowed_failures` -> is this because `call / opt` interpret the reported "failures" differently ?

Reply:
Yes